src/nemo-view.c

```diff
@@ -3133,6 +3135,7 @@ nemo_view_display_selection_info (NemoView *view)
 			}
 
 		}
+		view->details->detail_string = g_strdup (folder_item_count_str);
```

Comment:
Need to `g_free` any previous contents of `view->details->detail_string`.

It also needs to be freed in `nemo_view_finalize` (g_clear_pointer)